src/host.rs

```diff
@@ -88,6 +133,151 @@ impl Udp {
     }
 
     pub(crate) fn unbind(&mut self, addr: SocketAddr) {
-        assert!(self.binds.remove(&addr).is_some(), "unknown bind {}", addr)
+        assert!(self.binds.remove(&addr).is_some(), "unknown bind {}", addr);
+
+        trace!(?addr, protocol = %"UDP", "Unbind");
+    }
+}
+
+pub(crate) struct Tcp {
+    /// Bound server sockets
+    binds: IndexMap<SocketAddr, mpsc::UnboundedSender<(Syn, SocketAddr)>>,
```

Comment:
Why unbound?

I would put a limit on it, probably matching some linux standards, and note it's a TODO to make this configurable if need be.

Reply:
We could limit, but then we need to panic if the buffer is full. The network delivers into these channels and without some form of back pressure / flow control I'm not seeing how this would work. I recall we discussed punting back pressure a few weeks back. Do you think we should get something in now?